addons/mail/tests/common.py

```diff
@@ -663,23 +690,29 @@ def _assertMailMail(self, mail, recipients_list, status,
                         recipient,
                         **(email_values or {})
                     )
+            if email_to_all:
+                self.assertSentEmail(
+                    email_values['email_from'] if email_values and email_values.get('email_from') else author,
+                    email_to_all,
+                    **(email_values or {}))
 
     def assertMailMail(self, recipients, status,
                        email_to_recipients=None,
-                       mail_message=None, author=None,
+                       mail_message=None, author=None, email_to_all=None,
```

Comment:
Shouldn't we define email_to_all after recipients instead ? As the first line is the "main mail definition"